products/azure_rm_servicebusdisasterrecoveryconfig/api.yaml

```diff
@@ -0,0 +1,67 @@
+--- !ruby/object:Api::Product
+name: Azure Automation Variable
+prefix: arm
+versions:
+  - !ruby/object:Api::Product::Version
+    name: ga
+    base_url: NotUsedInAzure
+scopes:
+  - NotUsedInAzure
+azure_namespace: 'Microsoft.Automation'
+azure_version: 'stable/2015-10-31'
+objects:
+  - !ruby/object:Api::Resource
+    name: AutomationVariable
+    api_name: AutomationVariable
+    base_url: NotUsedInAzure
+
+    azure_sdk_definition: !ruby/object:Api::Azure::SDKDefinition
+      provider_name: Microsoft.Automation
+      create: !ruby/object:Api::Azure::SDKOperationDefinition
+        go_func_name: CreateOrUpdate
+        request:
+      read: !ruby/object:Api::Azure::SDKOperationDefinition
+        go_func_name: Get
+        request:
+        response:

```

Comment:
**Must Fix** Response is only required in Read operation: https://github.com/VSChina/magic-modules/blob/azure_backend/products/azservicebusrecovery/api.yaml#L56-L72.

Similar to the request, `go_type_name` and `go_field_name` is the field name in Go SDK generated from Autorest.